webapp/channels/src/components/admin_console/ldap_wizard/ldap_button_setting.tsx

```diff
@@ -33,8 +35,32 @@ const LDAPButtonSetting = (props: Props) => {
             });
             return;
         }
-        const successCallback = () => {
-            success?.();
+        const successCallback = (data?: LdapFilterTestResult[]) => {
+            // If this is the filter test button and we have results, pass them to the handler
+            if (props.setting.key === 'LdapSettings.TestFilters' && props.onFilterTestResults && data) {
+                props.onFilterTestResults(data);
+
+                const allTestsPassed = Array.isArray(data) && data.every((result) => result.error === '');
+                if (allTestsPassed) {
+                    success?.();
+                } else {
+                    const failedCount = data.filter((result) => result.error !== '').length;
+                    const totalCount = data.length;
+
+                    error({
+                        message: intl.formatMessage(
```

Comment:
these translations are all properly fixed in https://github.com/mattermost/mattermost/pull/31374